cache/lru_sized_cache.go

```diff
@@ -110,6 +117,10 @@ func (c *sizedLRU[K, V]) flush() {
 	c.currentSize = 0
 }
 
+func (c *sizedLRU[_, _]) len() int {
+	return c.elements.Len()
```

Comment:
Do we need the nil check here too?

Reply:
no - `elements` is initialized in `NewSizedLRU` here